tests/integration/test_config.py

```diff
@@ -282,3 +283,42 @@ def test_parse_special_tasks_families(flow, scheduler, validate, section):
             'foo(P1D)',
             'foot(P1D)'
         }
+
+
+def test_queue_treated_as_implicit(flow, validate, caplog):
+    """Tasks in queues but not in runtime generate a warning.
+
+    https://github.com/cylc/cylc-flow/issues/5260
+    """
+    reg = flow(
+        {
+            "scheduling": {
+                "queues": {"my_queue": {"members": "task1, task2"}},
+                "graph": {"R1": "task2"},
+            },
+            "runtime": {"task2": {}},
+        }
+    )
+    validate(reg)
+    assert (
+        'Queues contain tasks not defined in runtime'
+        in caplog.records[0].message
+    )
+
+
+def test_queue_treated_as_comma_separated(flow, validate):
+    """Tasks listed in queue should be separated with commas, not spaces.
+
+    https://github.com/cylc/cylc-flow/issues/5260
```

Comment:
Not sure both these links to the issue are needed?

Reply:
I think it's worth adding them to the tests they go with.